Sources/Common/Service/HttpRequestRunner.swift

```diff
@@ -6,7 +6,7 @@ import FoundationNetworking
 /**
  Exists to be able to mock http requests so we can test our HttpClient's response handling logic.
  */
-public protocol HttpRequestRunner: AutoMockable {
+internal protocol HttpRequestRunner: AutoMockable {
```

Comment:
comment makes it sound like this was a test helper just wanted to confirm that as changing `public`->`internal`